### PR TITLE
[11.0][MIG] mrp_bom_note

### PR DESCRIPTION
**Pull request by agungrachmatullah**
Migrating from v10 to v11 and adding unit test

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by agungrachmatullah** [comment]
@pedrobaeza i got this error in Travis-CI
image
do you have any idea for this one?

**Comment by LoisRForgeFlow** [comment]
Hi @agungrachmatulah 

Don't worry about that, just take care of the flase 8 issues: https://travis-ci.org/OCA/manufacture/jobs/368537220#L585-L590

Also, do not forget to squash commits as per @pedrobaeza suggestion.

Thanks for contributing!

**Comment by agungrachmatullah** [comment]
Hello @pedrobaeza , i have rebase my commit. is there any problem with my commits?

**Review by andhit-r — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I still insist in squashing together adjacent "OCA Transbot..." + auxiliar commits 

seleccion_003

**Comment by agungrachmatullah** [comment]
Thanks for the hints.
I was thinking if that commit no need to fixup since its under your commit --> c7775a

**Comment by agungrachmatullah** [comment]
Hello @pedrobaeza , is there anything missing from me?

**Comment by pedrobaeza** [comment]
You need 2 approvals from reviewers. You can review other PRs and ask in exchange that they review yours.

**Review by bimajatiwijaya — APPROVED** [review]

File: mrp_bom_note/README.rst
Comment (LoisRForgeFlow) [inline mrp_bom_note/README.rst]: Remove empty section

File: mrp_bom_note/README.rst
Comment (LoisRForgeFlow) [inline mrp_bom_note/README.rst]: 11.0

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Minor comments

**Review by LoisRForgeFlow — APPROVED** [review]
Functional review :+1:

**Pull request merged**
